docs/source/tutorial/registration.md

```diff
@@ -57,9 +57,13 @@ types of losses:
 ### Intensity based (image based) loss
 
 This type of loss measures the dissimilarity of the fixed image and warped moving image,
-which is adapted from the classical image registration methods. Intensity based loss is
-modality-independent and similar to many other well-studied computer vision and medical
-imaging tasks, such as image segmentation.
+which is an adaptation from classical image registration methods. Intensity based losses
+can perform poorly on multi-modality registrations (for example, SSD loss in CT-MRI
+registration). Generally, intensity based losses work best when there is an inherent
+consistency in appearance between moving and fixed images, which is more common in
+single-modality registration. Intensity based losses can be used with any modality and
+they complement other well-studied computer vision and medical imaging tasks, such as
```

Comment:
i feel after specific examples, "they complement other well-studied computer vision and medical imaging tasks..." can be removed.

Reply:
see previous change